tests/test_classical/test_agents.py

```diff
@@ -1,16 +1,16 @@
-import pytest
-import numpy as np
 import gym
+import numpy as np
+import pytest
 
-from genrl import QLearning, SARSA
+from genrl import SARSA, QLearning
 
 
 class TestAgents:
     def test_qlearning(self):
         env = gym.make("FrozenLake-v0")
         agent = QLearning(env)
 
-        assert np.any(agent.Q) == False
+        assert np.any(agent.Q) is not True
```

Comment:
Am I missing out on any edge cases here? Can someone take a look?

I've just changed `== False` to `is not True`

The reason I'm asking is because with just keeping `is False`, the test fails.

Reply:
It could be `None` too, that's not likely though. It's simpler to write `assert not np.any(agent.Q)`